tests/HOW_TO_RUN.md

```diff
@@ -0,0 +1,157 @@
+# Running tests
+
+Tests can be run from your favourite IDE, or from the command-line using Maven. Simply run the tests and the framework
+does the rest.
+
+## Configuring the test framework
+
+When running tests there are a few things than be configured:
+
+* Server type
+* Database type
+* Browser type
+
+There are a few options on how to configure the test framework, with the following ordinal:
+
+* System properties
+* Environment variables
+* `.env.test` file in the project hierarchy
+* A properties file specified with `kc.test.config` system property or `KC_TEST_CONFIG` environment variable
+
+For more details about the test framework, see the [README](../test-framework/README.md).
```

Comment:
Content copied to the HOW_TO_RUN.md and added this little reference to the test framework.